saos-persistence/src/main/java/pl/edu/icm/saos/persistence/content/JudgmentContentDeleter.java

```diff
@@ -0,0 +1,36 @@
+package pl.edu.icm.saos.persistence.content;
+
+import java.io.File;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author madryk
+ */
+@Service
+public class JudgmentContentDeleter {
```

Comment:
JudgmentContentFileDeleter? I think we should differentiate between text content in db and file content on disk.